ydb/core/mind/local.cpp

```diff
@@ -199,6 +199,10 @@ class TLocalNodeRegistrar : public TActorBootstrapped<TLocalNodeRegistrar> {
             }
             tabletAvailability->SetPriority(tabletInfo.Priority);
         }
+        const TString& slotName = AppData(ctx)->SlotName;
+        if (!slotName.Empty()) {
```

Comment:
Можно в одну строку:
```suggestion
        if (const TString& slotName = AppData(ctx)->SlotName; !slotName.Empty()) {
```